dev/src/reference.ts

```diff
@@ -388,13 +386,13 @@ export class DocumentReference {
    * A Precondition restricting this update can be specified as the last
    * argument.
    *
-   * @param {UpdateData|string|FieldPath} dataOrField - An object
-   * containing the fields and values with which to update the document
-   * or the path of the first field to update.
+   * @param {UpdateData|string|FieldPath} dataOrField An object containing the
+   * fields and values with which to update the document or the path of the
+   * first field to update.
    * @param {
-   * ...(*|string|FieldPath|Precondition)} preconditionOrValues -
-   * An alternating list of field paths and values to update or a Precondition
-   * to restrict this update.
+   * ...(*|string|FieldPath|Precondition)} preconditionOrValues An alternating
```

Comment:
Wow, your grep was even better than I imagined. 😀

Reply:
Clang format did this for me (in most places).